packages/react-native-payments/ios/Payments.mm

```diff
@@ -69,6 +69,29 @@ - (dispatch_queue_t)methodQueue
         }
     }
 
+    NSMutableArray *requiredBillingContactFields = [NSMutableArray array];
```

Comment:
Do we need to have the same implementation for the Android?

Reply:
We don't. We're able to do it all from the js side on Android. The changes in `packages/react-native-payments/src/class/payment-request/payment-request.ts` are all that's needed.